docs/integrations/builtin/credentials/hubspot.md

```diff
@@ -6,70 +6,89 @@ contentType: integration
 
 # HubSpot credentials
 
-You can use these credentials to authenticate the following nodes with HubSpot.
+You can use these credentials to authenticate the following nodes:
 
 - [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/)
 - [HubSpot Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.hubspottrigger/)
 
-/// note | API key deprecated
-HubSpot deprecated the API key authentication method. The option still appears in n8n, but you should use OAuth or APP Token.
-///
 ## Prerequisites
 
-Create a [HubSpot](https://www.hubspot.com/){:target=_blank .external-link} account.
+Create a [HubSpot](https://www.hubspot.com/){:target=_blank .external-link} account or [HubSpot developer](https://developers.hubspot.com/){:target=_blank .external-link} account.
 
-## Using OAuth
+## Supported authentication methods
 
-/// note | Note for n8n Cloud users
-You can skip these steps. Enter the credential name, then select **Connect my account** in the OAuth section to connect your HubSpot account to n8n.
+- API App token: Used with [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/) node
```

Comment:
```suggestion
- API app token: Used with [HubSpot](/integrations/builtin/app-nodes/n8n-nodes-base.hubspot/) node
```

Reply:
Should this be API app token, or just app token? In the interface it's just app token, and I wonder if calling it API app token could confuse users with the old deprecated API key credential?

Reply:
Oh that's an excellent idea for clarity!